civis/io/_tables.py

```diff
@@ -97,19 +99,22 @@ def read_civis(table, database, columns=None, use_pandas=False,
     """
     if use_pandas and NO_PANDAS:
         raise ImportError("use_pandas is True but pandas is not installed.")
+    if archive:
+        warnings.warn("`archive` is deprecated and will be removed in v2.0.0. "
+                      "Use `hidden` instead.", FutureWarning)
```

Comment:
Since this calls `read_civis_sql`, a user will get two deprecation warnings if they set `archive=True` in `read_civis`.  If it's removed here, the drawback is that the warning may be a bit more confusing since it will be raised in `read_civis_sql`.  What do you think is better?